.vscode/settings.json

```
@@ -63,9 +62,7 @@
   },
 
   // "terminal.integrated.defaultProfile.linux": "bash",
```

```suggestion
  "terminal.integrated.defaultProfile.linux": "bash",
```

yeah, iirc this option did not allow for fish atm,
but realistically (today on macOS, would actually
be `zsh` and on windows perhaps something other than
`fish`; `pwsh` or maybe `bash` somehow), looking now&hellip;

---

anyways on linux it is actually called `csh`, but we are not working on linux for the next week or so since container questions unanswered

---

> option did not allow for fish

this is about _our preference_ on linux (referring to our profile named `fish`)